lib/pina.rb

```diff
@@ -42,13 +43,22 @@ class Configuration
     attr_reader :api_version
 
     def initialize
+      clear
+    end
+
+    def clear
```

Comment:
Actually this method does not clear values. I would rename to `set_defaults` instead.